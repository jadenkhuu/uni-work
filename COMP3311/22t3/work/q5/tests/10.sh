python3 ./q5 "Warwick Farm" 2020-02-08
