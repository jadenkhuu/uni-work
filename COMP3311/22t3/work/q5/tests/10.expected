Race meeting at Warwick Farm on 2020-02-08

Race 1, prize pool $8000, run over 1000m
Flying Fox ridden by Javier Castellano wins $5600
Persimmon ridden by Glen Boss wins $1600
St Simon ridden by Liam Treadwell wins $800

Race 2, prize pool $7000, run over 1200m
The White Knight ridden by Emma-Jayne Wilson wins $4900
Hay Girl Hay ridden by Tommy Carberry wins $1400
Salmon Trout ridden by Wally Swinburn wins $700

Race 3, prize pool $7000, run over 2500m
Degeneres ridden by Jimmy Fortune wins $4900
Arnold Schwarze-neigh-ger ridden by Kaye Bell wins $1400
Victor Wild ridden by Michael Hussey wins $700

Race 4, prize pool $7000, run over 1700m
Sansovino ridden by Penny Ann Early wins $4900
Gluteus Maximus ridden by Brian Hernandez Jr. wins $1400
Hyperion ridden by Josephine Gordon wins $700

Eskimo Prince Stakes, prize pool $160000, run over 1200m
Spongebob Horsepant ridden by Patrick Husbands wins $112000
Justine Thyme ridden by Roy Higgins wins $32000
Shotover ridden by Alex Solis wins $16000
